neo/app/views/main.scala.html

```diff
@@ -17,23 +17,27 @@
   <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("lib/bootstrap-toggle/css/bootstrap-toggle.min.css")'>
   <link rel='stylesheet' href='@routes.Assets.versioned("stylesheets/main.css")'>
 
-  <script src='@routes.Assets.versioned("lib/angularjs/angular.min.js")'></script>
-  <script src='@routes.Assets.versioned("lib/json-bigint/dist/json-bigint.js")'></script>
-  <script src='@routes.Assets.versioned("lib/leaflet/leaflet.js")'></script>
-  <script src='@routes.Assets.versioned("lib/angular-leaflet-directive/angular-leaflet-directive.js")'></script>
-  <script src='@routes.Assets.versioned("lib/jquery/jquery.min.js")'></script>
-  <script src='@routes.Assets.versioned("lib/d3/d3.min.js")'></script>
-  <script src='@routes.Assets.versioned("lib/crossfilter/crossfilter.min.js")'></script>
-  <script src='@routes.Assets.versioned("lib/dc.js/dc.min.js")'></script>
-  <script src='@routes.Assets.versioned("lib/bootstrap/js/bootstrap.min.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/json-bigint/dist/json-bigint.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/leaflet/leaflet.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/angular-leaflet-directive/angular-leaflet-directive.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/jquery/jquery.min.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/d3/d3.min.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/crossfilter/crossfilter.min.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/dc.js/dc.min.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("lib/bootstrap/js/bootstrap.min.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("javascripts/lib/rbush.min.js")'></script>
+  <script type='text/javascript' src='@routes.Assets.versioned("javascripts/lib/turf.min.js")'></script>
   <script src='@routes.Assets.versioned("lib/bootstrap-toggle/js/bootstrap-toggle.min.js")'></script>
 
-  <script src='@routes.Assets.versioned("javascripts/app.js")'></script>
-  <script src='@routes.Assets.versioned("javascripts/map/controllers.js")'></script>
-  <script src='@routes.Assets.versioned("javascripts/common/services.js")'></script>
-  <script src='@routes.Assets.versioned("javascripts/timeseries/controllers.js")'></script>
-  <script src='@routes.Assets.versioned("javascripts/sidebar/controllers.js")'></script>
-  <script src='@routes.Assets.versioned("javascripts/searchbar/controllers.js")'></script>
+
```

Comment:
I was not a library added by me .It was added by @HotLemonJuice .This created conflict I just had to add it as it is in my PR